integrations/marketo-v2/lib/index.js

```diff
@@ -82,11 +82,16 @@ var apiNameMapping = {
 var Marketo = (module.exports = integration('Marketo V2')
   .assumesPageview()
   .global('Munchkin')
+  .global('MktForms2')
   .option('host', 'https://api.segment.io')
   .option('accountId', '')
   .option('projectId', '')
   .option('traits', [])
```

Comment:
Do we need to add the `marketoFormId` option somewhere here?

Reply:
Good call, I'm not sure but I will learn about this. There's also `marketoHostURL` and it should be added in here as well